docs/index.rst

```diff
@@ -3,6 +3,7 @@
   :hidden:
 
   gcloud-api
+  datastore-api
```

Comment:
`datastore-api` is misnamed now (it doesn't have any API documentation, and there is no `api` module to which it corresponds).